Modules/vc-module-store/VirtoCommerce.StoreModule.Web/JsonConverters/PolymorphicStoreJsonConverter.cs

```diff
@@ -0,0 +1,74 @@
+using System;
```

Comment:
This type hasn't registered anywhere, need to add registration 
https://github.com/VirtoCommerce/vc-platform-core/blob/master/VirtoCommerce.Platform.Web/Startup.cs#L79